include/bitcoin/server/define.hpp

```diff
@@ -41,12 +41,16 @@
 #define LOG_SERVER "server"
 
 // Avoid namespace conflict between boost::placeholders and std::placeholders.
-#define BOOST_BIND_NO_PLACEHOLDERS
+//#define BOOST_BIND_NO_PLACEHOLDERS
+using namespace std::placeholders;
```

Comment:
@evoskuil Hrm, strange.  I think this is a no-no, but it does resolve all builds.  After updating the boost version, the BOOST_BIND_NO_PLACEHOLDERS appears to not have the same effect it used to, as even local builds broke complaining about the placeholder issue.  The presence of that define appears to have no effect locally, so I've removed it.  If anyone has insight on this and/or the proper way to fix it, I'd like to know.